superdesk/media/image.py

```diff
@@ -39,7 +39,7 @@ def get_meta(file_stream):
             json.dumps(v)
             key = ExifTags.TAGS[k].strip()
 
-            if key == 'GPSInfo':
+            if key in 'GPSInfo':
```

Comment:
I wanted to upgrade to Pillow 3 but the metadata format changed  so I backpedaled :)